libbpf-tools/memleak.c

```diff
@@ -198,6 +204,7 @@ static const struct argp_option argp_options[] = {
 	{"max-size", 'Z', "MAX_SIZE", 0, "capture only allocations smaller than this size"},
 	{"obj", 'O', "OBJECT", 0, "attach to allocator functions in the specified object"},
 	{"percpu", 'P', NULL, 0, "trace percpu allocations"},
+	{"sym-pre", 'S', "SYM_PRE", 0, "memory alloctor symbols prefix"},
```

Comment:
Let's follow the python version and make this `symbols-prefix` instead.

Reply:
> Let's follow the python version and make this `symbols-prefix` instead.

done